.github/workflows/pr.yml

```diff
@@ -39,6 +39,44 @@ jobs:
     - build-wrappers
     with:
       build-docs: ${{ contains(github.head_ref, 'release') }}
+  publish-packages-to-sleet:
```

Comment:
Is this temporary?

Reply:
Didn't we say we wanted to publish the PR packages to sleet?

Reply:
Oh sorry, I must have not been clear here - I meant we wanted to do a one-off publish so that the server team can add a dependency on it for their e2e tests on CI. We don't need to do it on a continuous basis and definitely not for all PRs.